lib/galaxy/tools/deps/resolvers/modules.py

```diff
@@ -44,6 +45,8 @@ class ModuleDependencyResolver(DependencyResolver, MappableDependencyResolver):
     resolver_type = "modules"
 
     def __init__(self, dependency_manager, **kwds):
+        # Mapping file management
+        self._set_default_mapping_file(kwds)
```

Comment:
The next line should take care of this ☹️ , but the LMOD resolver does the same thing.

Reply:
@jmchilton what do you mean ? Do you want me to improve this. I can try. My contribution here has been only copy/paste :) so I can do better maybe !